### PR TITLE
Update: inputMode and keyboardType to allow other values

### PR DESCRIPTION
**Pull request by nitishxyz**
# Description

- Allowing the component to have other keyboard types as well, keeping numeric as default.

### CODE REVIEW

File: src/OtpInput/OtpInput.tsx [hunk 1]
```diff
@@ -104,6 +102,8 @@ export const OtpInput = forwardRef<OtpInputRef, OtpInputProps>((props, ref) => {
         testID="otp-input-hidden"
         onFocus={handleFocus}
         onBlur={handleBlur}
+        inputMode={textInputProps?.inputMode || "numeric"}
+        keyboardType={textInputProps?.keyboardType || "numeric"}
         {...textInputProps}
```
[the remark this answers is not in the document]
Comment (anday013) [reply src/OtpInput/OtpInput.tsx:107]: Did u get my point?

[the remark this answers is not in the document]
Comment (nitishxyz) [reply src/OtpInput/OtpInput.tsx:107]: Yes. Yet for some reason it's not working on our end. 

While using it this way works well. If you can verify it then you can close the PR. Thanks.

**Comment by anday013** [comment]
image
image

**Comment by anday013** [comment]
@nitishxyz take a look here

**Comment by nitishxyz** [comment]
> @nitishxyz take a look here

weird, i guess you can close the PR then, I know it should work but it isn't in our team's case for some reason, we've added patch for now, but I will try to debug the issue on my end. 

Thanks for testing it out though.

**Comment by anday013** [comment]
> > @nitishxyz take a look here
> 
> weird, i guess you can close the PR then, I know it should work but it isn't in our team's case for some reason, we've added patch for now, but I will try to debug the issue on my end.
> 
> Thanks for testing it out though.

make sure you are on the latest version of the library

**Pull request closed without merging**